modules/options/src/main/scala/scala/build/interactive/InteractiveFileOps.scala

```diff
@@ -0,0 +1,32 @@
+package scala.build.interactive
+
+object InteractiveFileOps {
+
+  def erasingPath(
+    interactive: Interactive,
+    printableDest: String,
+    destPath: os.Path
+  )(fallbackAction: () => Unit) = {
+    val msg = s"""|Destination path: $printableDest already exists.
+                  |Do you want to erasing $printableDest?""".stripMargin
+    val response = interactive.confirmOperation(msg)
+    response match {
+      case Some(true) => os.remove.all(destPath)
+      case _          => fallbackAction()
+    }
+  }
+
+  def addEntryToFile(
```

Comment:
`appendToFile` sounds clearer to me, given what this does. 
```suggestion
  def appendToFile(
```